docs/reference/manifest/requirements.md

```diff
@@ -1,15 +1,64 @@
 ---
 title: Requirements element in the manifest file
-description: The Requirements element specifies the minimum requirement set and methods your Office Add-in needs in order to activate.
-ms.date: 03/19/2019
+description: The Requirements element specifies the minimum requirement set and methods your Office Add-in needs to be activated by Office or to override base manifest settings.
+ms.date: 01/22/2022
 ms.localizationpriority: medium
 ---
 
 # Requirements element
 
-Specifies the minimum set of Office JavaScript API requirements ([requirement sets](../../develop/office-versions-and-requirement-sets.md#specify-office-applications-and-requirement-sets) and/or methods) that your Office Add-in needs to activate.
+The meaning of this element depends on whether it is used [in the base manifest](#in-the-base-manifest)] or [as a child of a **VersionOverrides** element](#as-a-child-of-a-versionoverrides-element).
 
-**Add-in type:** Content, Task pane, Mail
+> [!TIP]
+> Before using this element, be familiar with [Specify Office hosts and API requirements](../../develop/specify-office-hosts-and-api-requirements.md)
+
+## In the base manifest
+
+When used in the base manifest, that is, as a direct child of [OfficeApp](officeapp.md), the **Requirements** element specifies the minimum set of Office JavaScript API requirements ([requirement sets](../../develop/office-versions-and-requirement-sets.md#specify-office-applications-and-requirement-sets) and/or methods) that your Office Add-in needs to be activated by Office. The add-in will not be activated on any combination of Office version and platform (such as Windows, Mac, iOS, and web) that does not support the specified methods and requirement sets.
+
+**Add-in type:** Task pane, Mail
+
+## As a child of a VersionOverrides element
+
+When used as a child of [VersionOverrides](versionoverrides.md), specifies the minimum set of Office JavaScript API requirements ([requirement sets](../../develop/office-versions-and-requirement-sets.md#specify-office-applications-and-requirement-sets) and/or methods) that must be supported by the Office version and platform (such as Windows, Mac, iOS, and web) in order for the settings in the **VersionOverrides** element *that override base manifest settings* to take effect. If the requirements are not met, but the requirements specified in the base manifest *are* met (or there is no **Requirements** element in the base manifest), then the add-in will be activated, but the settings in the base manifest will be used, not the overriding settings in the **VersionOverrides**. However, child elements in the **VersionOverrides** *that support additional features, rather than override base manifest settings,* will still be applied unless:
```

Comment:
```suggestion
When used as a child of [VersionOverrides](versionoverrides.md), the **Requirements** element specifies the minimum set of Office JavaScript API requirements ([requirement sets](../../develop/office-versions-and-requirement-sets.md#specify-office-applications-and-requirement-sets) and/or methods) that must be supported by the Office version and platform (such as Windows, Mac, iOS, and web) in order for the settings in the **VersionOverrides** element *that override base manifest settings* to take effect. If the requirements aren't met, but the requirements specified in the base manifest *are* met (or there is no **Requirements** element in the base manifest), then the add-in will be activated, but the settings in the base manifest will be used, not the overriding settings in the **VersionOverrides**. However, child elements in the **VersionOverrides** *that support additional features, rather than override base manifest settings,* will still be applied unless:
```
This sentence has too many conjunctions (two "but" clauses) and doesn't make sense to me:

If the requirements aren't met, but the requirements specified in the base manifest *are* met (or there is no **Requirements** element in the base manifest), then the add-in will be activated, but the settings in the base manifest will be used, not the overriding settings in the **VersionOverrides**.

Reply:
```suggestion
When used as a child of [VersionOverrides](versionoverrides.md), specifies the minimum set of Office JavaScript API requirements ([requirement sets](../../develop/office-versions-and-requirement-sets.md#specify-office-applications-and-requirement-sets) and/or methods) that must be supported by the Office version and platform (such as Windows, Mac, iOS, and web) in order for the settings in the **VersionOverrides** element *that override base manifest settings* to take effect.

Consider an add-in specifies requirement A in the base manifest and it specifies requirement B inside the **VersionOverrides**. 

- If the platform and Office version don't support A, then the add-in isn't activated and Office doesn't even parse the **VersionOverrides** section of the manifest. 
- If both A and B are supported, then the add-in is activated and all the markup in the **VersionOverrides** takes effect. 
- If A is supported, but B is not, then the add-in is activated and *some* of the markup in the **VersionOverrides** takes effect. Specifically, child elements of the **VersionOverrides** that do not override base manifest elements take effect. For example, a **WebApplicationInfo** element takes effect. However, all child elements of the **VersionOverrides** that override a base manifest element, such as **Hosts**, do not take effect. Instead, Office uses the values of the base manifest markup that would otherwise have been overridden. 
```